proxy/http/HttpCacheSM.cc

```diff
@@ -36,11 +36,11 @@
 
 #define STATE_ENTER(state_name, event)                                                                                   \
   {                                                                                                                      \
-    REMEMBER(event, -1);                                                                                                 \
+    CACHE_REMAMBER(event, -1);                                                                                           \
     Debug("http_cache", "[%" PRId64 "] [%s, %s]", master_sm->sm_id, #state_name, HttpDebugNames::get_event_name(event)); \
   }
 
-#define REMEMBER(e, r) master_sm->add_history_entry(MakeSourceLocation(), e, r);
+#define CACHE_REMAMBER(e, r) master_sm->history.add_history_entry(MakeSourceLocation(), e, r);
```

Comment:
This is obviously a typo, but should be fixed: CACHE_REMAMBER vs CACHE_REMEMBER.